src/styles/mainObjectsForm.tsx

```diff
@@ -102,6 +104,46 @@ export const ButtonContainer = styled.div`
 
 // ------------------------------------------------------------
 
+const FOOTER_HEIGHT = 80
+const FOOTER_MARGIN = 80
+
+export const MainMinimumContent = styled.div`
```

Comment:
Those 3 classes are only used in one component right? Was wondering why putting them in a global style declaration file